### DISCUSSION TITLE
Id on objects

### DISCUSSION THREAD

**Discussion by MarcoNais** [YetiForceCompany/YetiForceCRM#15779] [How to] [answered]
Hi, I am testing YetiForce, when I create a Lead, opportunity or proposal is it possible to mark it with a progressive ID?
Because it can make it more effective identify the objects.
For Example
![image_2022-03-08_105434](https://user-images.githubusercontent.com/101178986/157212485-8ca3175b-00ca-4e65-832a-dff175a1e221.png)

Mark the first row as LD001 and the second as LD002, in order to have an identification for Lead and the other object in the CRM (Project, Opportunity and so long)

**Comment by rskrzypczak** [comment, marked as answer]
I don't really know what you mean, each module has a Number field in its structure, the field is automatically populated when the record is created.
![image](https://user-images.githubusercontent.com/10221999/157225142-982ae8e2-aeb5-4dde-acca-d1777dbfc216.png)

Custom settings panel: _Software configuration/Standard modules/Modules - Prefixes_

**Reply by MarcoNais** [reply]
Great, I hadn't found the field!
Is it possible to edit the leads dashboard to display next to the name?

**Reply by mariuszkrzaczkowski** [reply]
https://gitdeveloper.yetiforce.com/index.php?module=Search&parent=Settings&view=Index
![image](https://user-images.githubusercontent.com/10198654/157227194-00195dea-082a-4ab6-8e36-ff728ca959a0.png)

**Reply by MarcoNais** [reply]
Great, thanks!
